### PR TITLE
Refugee Center's Surface Refugee Mission Descriptions

### PR DESCRIPTION
**Pull request by gettingusedto**
#### Summary
None

#### Purpose of change
Some of the Refugee Center's surface refugee missions could do with mission descriptions.

#### Describe the solution
Added descriptions to:

Alonso
- Alonso must have his special pants

Boris
- Bring Boris a circular saw
- Bring Boris his son's lost writing

Jenny
- Find reference material about air rifles for Jenny
- Find a gunsmithing textbook

Random NPC Mission
- Retrieve Software

#### Describe alternatives you've considered

Not doing so and let it be confusing and vague.

#### Additional context

As usual, corrections would be appreciated.

### CODE REVIEW

File: data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json [hunk 1]
```diff
@@ -579,6 +579,7 @@
     "id": "MISSION_REFUGEE_Jenny_GET_PNEUMATICS",
     "type": "mission_definition",
     "name": { "str": "Find reference material about air rifles for Jenny." },
+    "description": "Find reference material about air rifle.  <color_yellow>A book that has 'Pneumatic' in its name</color> could be the thing she wants.",
```
Comment (Karol1223) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json:582]:
```suggestion
    "description": "Find reference material about air rifles.  <color_yellow>A book that has 'Pneumatic' in its name</color> could be the thing she wants.",
```

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json [hunk 2]
```diff
@@ -579,6 +579,7 @@
     "id": "MISSION_REFUGEE_Jenny_GET_PNEUMATICS",
     "type": "mission_definition",
     "name": { "str": "Find reference material about air rifles for Jenny." },
+    "description": "Find reference material about air rifles.  <color_yellow>A book that has 'Pneumatic' in its name</color> could be the thing she wants.",
```
Comment (Xaleth) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json:582]:
```suggestion
    "description": "Find reference material about air rifles.  <color_yellow>A book that has 'Pneumatic' in its name</color_yellow> could be the thing she wants.",
```

File: data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json [hunk 3]
```diff
@@ -661,6 +662,7 @@
     "id": "MISSION_REFUGEE_Jenny_GET_TEXT",
     "type": "mission_definition",
     "name": { "str": "Find a gunsmithing textbook" },
+    "description": "Find a gunsmithing textbook for Jenny.  <color_yellow>A book with 'Gunsmithing' and/or 'Firearm Design' in its name</color> will do.",
```
Comment (Xaleth) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json:665]:
```suggestion
    "description": "Find a gunsmithing textbook for Jenny.  <color_yellow>A book with 'Gunsmithing' and/or 'Firearm Design' in its name</color_yellow> will do.",
```

File: data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json (hunk 2 above)
Comment (gettingusedto) [reply data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json:582]: </color> works just fine. dont need to add the yellow again.

File: data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json (hunk 3 above)
Comment (gettingusedto) [reply data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json:665]: Same thing, `</color>` works fine here

**Pull request merged**